core/pipeline/label.go

```diff
@@ -64,7 +64,7 @@ func loadRootLabels(workdir string) []Label {
 // ServiceHostnameLabel is annotated on all services started by Dagger.
 const ServiceHostnameLabel = "dagger.io/service.hostname"
 
-func loadGitLabels(workdir string) ([]Label, error) {
+func LoadGitLabels(workdir string) ([]Label, error) {
```

Comment:
is this uppercased so `label_test.go` can access it?

I'm thinking why `label_test.go` is under `pipeline_test`  and not `pipeline` package :thinking:

Reply:
Old habits. I tend to put tests in a separate package to focus them on external interfaces, but really all of this code is internal, and part of it was exposed already, so I just exposed more. I don't feel very strongly about this tho.